tools/go.mod

```diff
@@ -21,10 +21,11 @@ require (
 
 require (
 	github.com/beorn7/perks v1.0.1 // indirect
-	github.com/cespare/xxhash/v2 v2.1.2 // indirect
+	github.com/cespare/xxhash/v2 v2.2.0 // indirect
 	github.com/dgryski/go-rendezvous v0.0.0-20200823014737-9f7001d12a5f // indirect
 	github.com/fsnotify/fsnotify v1.4.9 // indirect
 	github.com/gdamore/encoding v1.0.0 // indirect
+	github.com/go-redis/redis/v8 v8.11.4 // indirect
```

Comment:
v8 still being referenced in the tools folder.

Reply:
Please fix

Reply:
@l0nax please fix.

Check module used with cmd: `go mod why github.com/go-redis/redis/v8`